lib/Runtime/Language/InterpreterStackFrame.cpp

```diff
@@ -7933,6 +7933,34 @@ const byte * InterpreterStackFrame::OP_ProfiledLoopBodyStart(const byte * ip)
         return OP_LdObjSlot(slotArray, playout->SlotIndex2);
     }
 
+    template <class T>
+    Var InterpreterStackFrame::OP_LdModuleSlot(Var instance, const unaligned T* playout)
+    {
+        Js::SourceTextModuleRecord* moduleRecord = scriptContext->GetLibrary()->GetModuleRecord(playout->SlotIndex1);
+        Assert(moduleRecord != nullptr);
+
+        Var* moduleRecordSlots = moduleRecord->GetLocalExportSlots();
+        Assert(moduleRecordSlots != nullptr);
+
+        Var value = moduleRecordSlots[playout->SlotIndex2];
+        Assert(value != nullptr);
+
+        return value;
+    }
+
+    inline void InterpreterStackFrame::OP_StModuleSlot(Var instance, int32 slotIndex1, int32 slotIndex2, Var value)
+    {
+        Assert(value != nullptr);
+
+        Js::SourceTextModuleRecord* moduleRecord = scriptContext->GetLibrary()->GetModuleRecord(slotIndex1);
+        Assert(moduleRecord != nullptr);
+
+        Var* moduleRecordSlots = moduleRecord->GetLocalExportSlots();
+        Assert(moduleRecordSlots != nullptr);
+
+        moduleRecordSlots[slotIndex2] = value;
```

Comment:
I'm guessing that an assert should be sufficient here, since the index is computed internally and not provided by the user.